{
  try {
    resolver.resolve(module,artifactType,result);
  }
 catch (  Throwable t) {
    throw new AssertionError(String.format("Unexpected exception when resolving artifacts of type %s for component %s",artifactType.getName(),module.getComponentId()),t);
  }
}
